src/components/slider.tsx

```diff
@@ -0,0 +1,88 @@
+import { getTrackBackground, Range } from "react-range"
+import React, { FC, useState } from "react"
+
+import InputFilter from "./filters/input"
+import Label from "./fieldHelpers/label"
+import colors from "../tailwind/colors"
+
+interface Props {
+  step: number
+  min: number
+  max: number
+  name: string
+  defaultValue: number
+  inputLabel: string
+  required?: boolean
+}
+
+const Slider: FC<Props> = ({
+  step,
+  min,
+  max,
+  name,
+  defaultValue,
+  inputLabel,
+  required = false,
+}) => {
+  const [rangeValues, setRangeValues] = useState([defaultValue])
+  const [inputValue, setInputValue] = useState(defaultValue)
```

Comment:
Is it maybe possible to remove this state variable and use `String(rangeValues[0])` instead for the input field?